openshift/gabi.template.yaml

```diff
@@ -68,6 +68,8 @@ objects:
           - --provider=openshift
           - --openshift-service-account=${GABI_INSTANCE}
           - --upstream=http://localhost:8080
+          # Reserved for future use.
+          # - --upstream-timeout=${OAUTH_PROXY_UPSTREAM_TIMEOUT}
```

Comment:
**To the reviewer:**

Support for setting upstream timeout will be added to Red Hat's fork of the OAuth2 Proxy shortly.